airbyte-integrations/connectors/source-smartsheets/sample_files/catalog.json

```
@@ -0,0 +1,43 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "aws_s3_sample",
+        "supported_sync_modes": ["full_refresh"],
+        "source_defined_cursor": false,
+        "source_defined_primary_key": [["id"]],
+        "json_schema": {
```

I agree with Maksym here that the `json_schema` field can be an empty object now.

```suggestion
        "json_schema": {}
```